page02_playlist.php

```diff
@@ -1,12 +1,13 @@
 <?php
```

Comment:
same changes as query 1 and 2 - since code was copy and pasted but change in queries